### PR TITLE
BUILD-759 Move Jenkins + Travis QA to Cirrus CI

### PR DESCRIPTION
**Pull request by wouter-admiraal-sonarsource**
Old job on cix: https://cix.sonarsource.com/job/sonar-scanner-cli-qa/

### CODE REVIEW

File: .cirrus.yml [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+
+#
+# TASKS
+#
+build_task:
+  gke_container:
+    <<: *CONTAINER_DEFINITION
+    cpu: 2
+    memory: 2G
```
Comment (julienlancelot) [inline .cirrus.yml]: You could try to reduce cpu/memory and check timing remains identical

File: .cirrus.yml [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+    builder_image_name: docker-builder-v2
+    cluster_name: cirrus-uscentral1a-cluster
+    zone: us-central1-a
+    namespace: default
+    cpu: 1
+    memory: 2G
+  env:
+    matrix:
+      - SQ_VERSION: LATEST_RELEASE[7.9]
+      - SQ_VERSION: DEV
```
Comment (julienlancelot) [inline .cirrus.yml]: On CIX, it was also running on DOGFOOD, it's no more needed ?

**Comment by julienlancelot** [comment]
You've forgotten to remove the JenkinsFile

File: .cirrus.yml (hunk 2 above)
Comment (wouter-admiraal-sonarsource) [reply .cirrus.yml]: No, discussed with Simon. We drop it. Same for macOS QA's (not supported by Google Compute Engine).

**Pull request merged**